config.go

```diff
@@ -0,0 +1,226 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/go-utils/retry"
+	"github.com/bitrise-io/go-xcode/appleauth"
+	"github.com/bitrise-io/go-xcode/devportalservice"
+	"github.com/kballard/go-shellquote"
+)
+
+// Config contains inputs parsed from environment variables
+type Config struct {
+	WorkDir string `env:"work_dir,dir"`
+	Lane    string `env:"lane,required"`
+
+	BitriseConnection   string          `env:"connection,opt[automatic,api_key,apple_id,off]"`
+	AppleID             string          `env:"apple_id"`
+	Password            stepconf.Secret `env:"password"`
+	AppSpecificPassword stepconf.Secret `env:"app_password"`
+	APIKeyPath          stepconf.Secret `env:"api_key_path"`
+	APIIssuer           string          `env:"api_issuer"`
+
+	UpdateFastlane bool `env:"update_fastlane,opt[true,false]"`
+	VerboseLog     bool `env:"verbose_log,opt[yes,no]"`
+	EnableCache    bool `env:"enable_cache,opt[yes,no]"`
+
+	GemHome string `env:"GEM_HOME"`
+
+	// Used to get Bitrise Apple Developer Portal Connection
+	BuildURL        string          `env:"BITRISE_BUILD_URL"`
+	BuildAPIToken   stepconf.Secret `env:"BITRISE_BUILD_API_TOKEN"`
+	AuthInputs      appleauth.Inputs
+	AuthCredentials appleauth.Credentials
+	LaneOptions     []string
+}
+
+// ProcessConfig ...
+func (s FastlaneRunner) ProcessConfig() (Config, error) {
```

Comment:
nit: `s` is probably a leftover from the previous state, I would call it `f` or `s` in this context.

Reply:
I went with `s` as a short for `step` (I'm assuming). The xcode-archive step, which I used as a blueprint, also uses this convention:
```
func (s XcodebuildArchiver) ProcessInputs() (Config, error)
```

Reply:
Oh, then this line in  xcode-archive also slipped through code review 😅 

We follow the official Go coding conventions, which has a section about receiver names: https://github.com/golang/go/wiki/CodeReviewComments#receiver-names